README.md

```diff
@@ -1,2 +1,10 @@
 # training-notebooks
- A set of Notebooks to demonstrate how to access the images and metadata from OMERO
+
+ A set of Notebooks to demonstrate how to access the images and metadata from OMERO.
+
+ The notebooks in this repository are meant to exemplify how to access data in OMERO.
+
+ | **Notebook** | **Lang** | **Description** |
+ | **[OMERO example](Python/OMEROHelloWolrdNotebook.ipynb)** | Py | Shows how to connect to OMERO and read data. |
+ | **[Calculate Sharpness](Python/CalculateSharpness.ipynb)** | Py | Calculates sharpness of images and generates heatmaps. |
+ | **[Illumination Correction](Python/CalculateSharpness.ipynb)** | Py | Correct the selected image. |
```

Comment:
The table isn't formatted correctly:
<img width="915" alt="screen shot 2018-05-07 at 10 01 12" src="https://user-images.githubusercontent.com/1644105/39693973-a4ca582e-51dd-11e8-8679-650548857a17.png">

Try http://www.tablesgenerator.com/markdown_tables ?

Reply:
Need to go back to that